downloadDirectory/src/test/java/downloadDirectory/TestProtegeExporter.java

```diff
@@ -0,0 +1,67 @@
+package downloadDirectory;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Properties;
+
+import org.gk.persistence.MySQLAdaptor;
+import org.junit.Before;
+import org.junit.Test;
+import org.reactome.release.downloadDirectory.ProtegeExporter;
+
+@SuppressWarnings("static-method")
+public class TestProtegeExporter
+{
+	private static final String RELEASE_NUM = "70";
+	private static String host;
+	private static String name;
+	private static String user;
+	private static String password;
+	static final String PATH_TO_JAVA_ROOT = "/home/sshorser/workspaces/reactome/ReleaseNG/data-release-pipeline/downloadDirectory/";
+	
+	@Before
+	public void setup() throws FileNotFoundException, IOException
+	{
+		Properties props = new Properties();
+		try(FileInputStream fis = new FileInputStream("src/test/resources/db.properties"))
+		{
+			props.load(fis);
+			TestProtegeExporter.host = props.getProperty("db.host");
+			TestProtegeExporter.name = props.getProperty("db.name");
+			TestProtegeExporter.user = props.getProperty("db.user");
+			TestProtegeExporter.password = props.getProperty("db.password");
+			String release = RELEASE_NUM;
+			Files.createDirectories(Paths.get(release+"/"));
```

Comment:
Can there be a corresponding "After" to clean up this created directory from a run of the tests?

Reply:
I was going to, but I didn't because I also wanted to be able to inspect the file manually after the code finished.

Reply:
Okay, I see.  Is there any issue if the test is re-run with the directory and files already existing?  Once the code is finished development, though, the test code should probably clean up after itself?  Any checks done manually could be added as tests in their own right.

Reply:
I might consider building a "cleanup" option that can be triggered by users who want that...